### PR TITLE
Add cache, hopefully speed some things up

### PR DESCRIPTION
**Pull request by maplant**

### CODE REVIEW

File: mobile_verifier/migrations/21_index_hex_coverage.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS hex_coverage_radio_key_idx ON hex_coverage USING btree (radio_key)
```
Comment (jeffgrunewald) [inline mobile_verifier/migrations/21_index_hex_coverage.sql:1]: Isn’t btree the default? Just looking for clarity?

**Review by bbalser — APPROVED** [review]

File: mobile_verifier/src/heartbeats/cbrs.rs [hunk 2]
```diff
@@ -79,21 +80,27 @@ impl HeartbeatDaemon {
...
 
-    async fn process_file(
-        &self,
+    async fn process_file<'a>(
+        &'a self,
         file: FileInfoStream<CbrsHeartbeatIngestReport>,
-        heartbeat_cache: &Cache<(String, DateTime<Utc>), ()>,
-        coverage_claim_time_cache: &CoverageClaimTimeCache,
-        covered_hex_cache: &CoveredHexCache,
+        heartbeat_cache: &Arc<Cache<(String, DateTime<Utc>), ()>>,
```
Comment (jeffgrunewald) [inline mobile_verifier/src/heartbeats/cbrs.rs:85]: Why the reference? I thought arcs were cheap to copy

**Review by jeffgrunewald — APPROVED** [review]

File: mobile_verifier/migrations/21_index_hex_coverage.sql (hunk 1 above)
Comment (bbalser) [reply mobile_verifier/migrations/21_index_hex_coverage.sql:1]: yes, I just copied the command that was generated by my sql client

File: mobile_verifier/src/heartbeats/cbrs.rs (hunk 2 above)
Comment (maplant) [reply mobile_verifier/src/heartbeats/cbrs.rs:85]: Fewer lines of code changed :-/

**Pull request merged**